src/Symfony/Bundle/FrameworkBundle/Resources/config/schema/symfony-1.0.xsd

```diff
@@ -1066,4 +1066,22 @@
         <xsd:attribute name="enabled" type="xsd:boolean" />
     </xsd:complexType>
 
+    <xsd:complexType name="type-info" mixed="true">
+        <xsd:sequence>
+            <xsd:element name="aliases" minOccurs="0" maxOccurs="1">
+                <xsd:complexType>
+                    <xsd:sequence>
+                        <xsd:element name="alias" minOccurs="0" maxOccurs="unbounded">
+                            <xsd:complexType>
+                                <xsd:attribute name="name" type="xsd:string" use="required"/>
+                                <xsd:attribute name="type" type="xsd:string" use="required"/>
```

Comment:
this should use the body of the tag instead, to be consistent with our other scalar prototypes in XML config files (and not requiring special handling in the Configuration class)